google/cloud/aiplatform/prediction/model_server.py

```diff
@@ -186,6 +187,10 @@ def set_number_of_workers_from_env() -> None:
 
 
 if __name__ == "__main__":
+    cpr_user_dir_name = os.getenv("CPR_USER_DIR_NAME")
+    if cpr_user_dir_name:
+        sys.path.append(os.path.join(os.getcwd(), cpr_user_dir_name))
```

Comment:
Can we insert at the beginning of sys.path? So it searches the user's modules first.

Reply:
Done.